src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -43,6 +42,8 @@ public sealed class Cookie
         internal const int MaxSupportedVersion = 1;
         internal const string MaxSupportedVersionString = "1";
 
+        internal const string NameAttributeName = "Name";
+        internal const string ValueAttributeName = "Value";
```

Comment:
We shouldn't be making product source changes in a PR and commit about tests.  At the very least, such changes should be separated out into a commit that appropriately describes the changes being made.